### ISSUE TITLE
Tensorflow install error

### ISSUE THREAD

**Issue by bharath-r17** [nicknochnack/TFODCourse#15]
ERROR: Could not find a version that satisfies the requirement tensorflow (from versions: none)
ERROR: No matching distribution found for tensorflow
getting the following error when i try to pip install tensorflow

**Comment by BeBrogs** [comment]
Create virtual env, activate then clone again. Worked for me

**Comment by Akshaybhat5** [comment]
Is that you're on Mac with an M1 chip?

**Comment by YavinOwens** [comment]
> Is that you're on Mac with an M1 chip?

im having this issue im guessing this is because i have the m1 chip?

**Comment by sholaremu** [comment]
Im having issue on setup path. Tensorflow file didn't appear on my ANPR directory. Please any help on this?

**Comment by Akshaybhat5** [comment]
Yes, I'm on Mac with an M1 chip. After few hours of browsing here and there
on the internet, I got this message;- 'Tensorflow can be installed on the
Mac but still not flexible like in an intel based chips'.
